region/norfair/east.json

```diff
@@ -179,31 +186,32 @@
                   "heatProof",
                   "canHeatRun"
                 ]},
-                {"or":[
-                  {"and":[
+                {"cost": [
+                  {"heatFrames": 100}
+                ]},
+                {"obstacle":{
+                  "id": "A",
```

Comment:
I don't see in what situation the obstacle is useful here?

Reply:
The goal was to put a hint in there that these requirements are centered around killing enemies, but maybe I'm jumping the gun. This can be done later if deemed necessary. I'll take it out.